internal/datastore/postgres/reader.go

```diff
@@ -252,6 +264,9 @@ func (r *pgReader) LookupNamespacesWithNames(ctx context.Context, nsNames []stri
 		return r.aliveFilter(original).Where(clause)
 	})
 	if err != nil {
+		if wrappedErr := pgxcommon.WrapMissingTableError(err); wrappedErr != nil {
+			return nil, wrappedErr
```

Comment:
```suggestion
			err = wrappedErr
```

otherwise you lose the `errUnableToListNamespaces` below